vendor.conf

```diff
@@ -50,8 +50,8 @@ github.com/hashicorp/go-sockaddr                    6d291a969b86c4b633730bfc6b8b
 github.com/hashicorp/go-multierror                  fcdddc395df1ddf4247c69bd436e84cfa0733f7e
 github.com/hashicorp/serf                           598c54895cc5a7b1a24a398d635e8c0ea0959870
 github.com/docker/libkv                             458977154600b9f23984d9f4b82e79570b5ae12b
-github.com/vishvananda/netns                        604eaf189ee867d8c147fafc28def2394e878d25
-github.com/vishvananda/netlink                      b2de5d10e38ecce8607e6b438b6d174f389a004e
+github.com/vishvananda/netns                        13995c7128ccc8e51e9a6bd2b551020a27180abd
+github.com/vishvananda/netlink                      a2ad57a690f3caf3015351d2d6e1c0b95c349752 # v1.0.0
```

Comment:
@thaJeztah FYI I just realized that this is not v1.0.0. Maybe we want to keep that in mind for cherry-picking.

Reply:
I just double checked,  but looks like this commit matches the v1.0.0 tag? https://github.com/vishvananda/netlink/commit/a2ad57a690f3caf3015351d2d6e1c0b95c349752

Reply:
omg... they have a v1.0.0 branch that's not the tag...

Reply:
opened https://github.com/vishvananda/netlink/issues/465

Reply:
oooooooh!

I wonder which one `vndr` picks if we don't specify the sha 🤔